docs/Analyzer Configuration.md

```diff
@@ -141,6 +141,17 @@ Default Value: `NonExceptionPaths`.
 
 Example: `dotnet_code_quality.dispose_analysis_kind = AllPaths`
 
+#### Configure dispose ownership transfer for arguments passed to constructor invocation
```

Comment:
An example of code that this applies to would be nice.

Reply:
Sure, I am about to send another trivial PR about namespace name changes, will add an example for this too. Thanks!